openpype/scripts/ocio_wrapper.py

```diff
@@ -174,5 +174,79 @@ def _get_views_data(config_path):
     return data
 
 
+def _get_display_view_colorspace_name(config_path, display, view):
+    """Returns the colorspace attribute of the (display, view) pair.
+
+    Args:
+        config_path (str): path string leading to config.ocio
+        display (str): display name e.g. "ACES"
+        view (str): view name e.g. "sRGB"
+
+
+    Raises:
+        IOError: Input config does not exist.
+
+    Returns:
+        view color space name (str) e.g. "Output - sRGB"
+    """
+
+    config_path = Path(config_path)
+
+    if not config_path.is_file():
+        raise IOError("Input path should be `config.ocio` file")
+
+    config = ocio.Config().CreateFromFile(str(config_path))
```

Comment:
```suggestion
    config = ocio.Config.CreateFromFile(str(config_path))
```
Isn't [`CreateFromFile` a static method on the config class](https://opencolorio.org/old/developers/bindings/PythonAPI.html#PyOpenColorIO.Config.CreateFromFile)? And thus you don't need an instance to run it on. In your case you're creating an OCIO instance - and then generating a new instance from the class method.

I notice the same `CreateFromFile()` call is done on `ocio.Config()` across the code above but there shouldn't be any need to initialize that empty instance.